app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics-controllers");
+
+app.use(express.json());
```

Comment:
This line of code isn't needed until we post a comment. Please be careful to only include lines of code we need for the ticket in hand